_snippets/glusterfs.adoc

```diff
@@ -0,0 +1,27 @@
+////
+How to use this file:
+
+* Use `include::_snippets/glusterfs.adoc[]` to enable the variables on a given
+  page.
+* When discussing the storage technology itself, just use "GlusterFS", e.g.
+  "GlusterFS volumes". It is incorrect to say, for example, "Red Hat Gluster
+  Storage volumes".
+* When discussing the base project or product, use `{gluster}`.
+* When discussing configurations of {gluster} that are managed by heketi, use
+  `{gluster-native}` if {gluster} is containerized and `{gluster-external}` if
+  it is not containerized.
```

Comment:
Because `{gluster-native}` and `{gluster-external}` change depending on whether the docs ar ebeing rendered for origin or enterprise. There is no such thing as "Container-Native Storage" in the upstream, it is purely a downstream product. Same for "Container-Ready Storage"

Reply:
Theses are downstream docs. Can we just maintain a delta?

Reply:
These are not downstream docs. These apply to both OCP and Origin, and Origin is technically upstream. Thus, any reference to downstream products in Origin documentation would be a bug.